puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/rubygemRPMNightlyRelease.groovy

```diff
@@ -0,0 +1,31 @@
+pipeline {
+    agent { label 'rpmbuild' }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage ("Build Gem") {
+            steps {
+                dir('gem-build') {
+                    git(url: "https://github.com/${owner_repo}", branch: branch)
+                    sh("gem build ${gemspec}")
+                    archiveArtifacts(artifacts: '*.gem')
```

Comment:
It's important we clean any old gems or we can archive multiple gems this way. Do I miss the cleanup step?

Reply:
Agreed, we should have a post {} section to deleteDir at the end

Reply:
can do!